src/installer.ts

```diff
@@ -80,6 +85,26 @@ export async function installBuf(
   return [binName, resolvedVersion];
 }
 
+// resolvePublicGitHub returns a GitHub instance for the public GitHub API if
+// the environment is a GitHub Enterprise instance.
+function resolvePublicGitHub(
+  github: InstanceType<typeof GitHub>,
+  inputs: Inputs,
+): InstanceType<typeof GitHub> {
+  const apiUrl = process.env.GITHUB_API_URL || ``;
```

Comment:
there isn't some API we can call to get the URL the github object will use?

Reply:
Typescript complains with: `github.request.defaults.baseUrl`, but I believe that would be the way to check.